decisions/0010-testing-the-editor-pipeline.md

```diff
@@ -0,0 +1,39 @@
+# 10. Testing the Editor Pipeline
+
+Date: 2021-05-26
+
+## Status
+
+Accepted
+
+## Context
+
+Our pipeline follows a continuous delivery method with the manual gate to release to production.
+
+![alt text](0010-release-pipeline-context.png "Diagram of how the editor is released through the pipeline.")
+
+We have one Editor in test which causes issues when the rest of the team want to test a feature. Also, while the Editor is tied up in testing the pipeline is blocked for deploying to live-production.
+
+## Decision
+
+![alt text](0010-release-pipeline-proposed.png "Diagram of how the editor is released through the pipeline to have a testable editor.")
+
+The main problem to solve is having a feature that is being developed available to the wider team to test. Running locally was not considered as the MOJ Macbooks are locked down to those outside of the technical disciplines.
+
+Changing the pipeline to create an Editor (all plugged in to the dev and test environments). This will involve, building and deploying the editor code into a pod and attaching to a unique URL. Not every branch will need to create a new test editor, the developer will need to add "testable-" to the beginning of the branch name (i.e. `testable-my-really-awesome-feature`).
```

Comment:
one potential avenue:

```
filters:
  branches:
  only: /testable-*/
```

branch filtering is apparently possible in circleci